gpkit/tests/t_model.py

```diff
@@ -354,6 +355,23 @@ def test_signomials_not_allowed_in_objective(self):
             with self.assertRaises(TypeError):
                 sol = m.localsolve(verbosity=0)
 
+    def test_breakdown(self):
```

Comment:
this should not be inside the `TestSP` class -- make a separate `TestBreakdown` class, separate from `TestGP` and `TestSP`